client/ayon_deadline/scripts/remote_publish.py

```diff
@@ -0,0 +1,45 @@
+import os
+import pyblish.api
+import pyblish.util
+
+from ayon_core.lib import Logger
+from ayon_core.pipeline.create import CreateContext
+from ayon_core.pipeline import registered_host
+
+
+def remote_publish(log):
+    error_format = "Failed {{plugin.__name__}}: {{error}}:{{error.traceback}}"
+    host = registered_host()
+    create_context = CreateContext(host)
+    pyblish_context = pyblish.api.Context()
+    solo_instance_ids: set[str] = set(
+        os.environ.get("instance_ids", "").split(";")
```

Comment:
Should this be all caps, because it's an env var?
```suggestion
        os.environ.get("INSTANCE_IDS", "").split(";")
```

And maybe even use `os.getenv`? @iLLiCiTiT ?